tool-plugins/intellij/README.md

```diff
@@ -1,57 +1,89 @@
+
 # Ballerina plugin for IntelliJ IDEA
 
 [![Version](https://img.shields.io/jetbrains/plugin/v/9520-ballerina.svg)](https://plugins.jetbrains.com/plugin/9520-ballerina)
 [![Downloads](https://img.shields.io/jetbrains/plugin/d/9520-ballerina.svg)](https://plugins.jetbrains.com/plugin/9520-ballerina)
 
-## Building from the source
-
-Run `./gradlew buildPlugin`. In the **build/distributions** directory, **ballerina-intellij-idea-plugin-[VERSION].zip** will be created.
-
-**Note:** On Windows, you don’t need the leading `./` in front of the `gradlew` command
-
-## Installing the plugin to IDEA
+[Ballerina](https://ballerina.io/) is a simple programming language whose syntax and runtime address the hard 
+problems of integration. As a Turing complete language, Ballerina enables agility with edit, build, and run cycles. 
+Ballerina code is compiled into services that include transactions, embedded brokers and gateway runtime.
+Ballerina makes it easy to build resilient services that integration and orchestrate across distributed endpoints. 
```

Comment:
Change to "Ballerina makes it easy to build resilient services for integration and orchestrate across distributed endpoints."